Hammerspoon/setup.lua

```diff
@@ -26,12 +26,14 @@ hs.configdir = configdir
 --- A string containing the full path to the `docs.json` file inside Hammerspoon's app bundle. This contains the full Hammerspoon API documentation and can be accessed in the Console using `help("someAPI")`. It can also be loaded and processed by the `hs.doc` extension
 hs.docstrings_json_file = docstringspath
 
---- hs.showError(err)
+--- hs.showError(err,lvl,terminate)
 --- Function
 --- Shows an error to the user, using Hammerspoon's Console
 ---
 --- Parameters:
 ---  * err - A string containing an error message
+---  * lvl - A number containing the level in the call stack for debug.traceback
```

Comment:
lvl and terminate should say that they're optional, and what their defaults are :)